src/main/java/org/jabref/gui/actions/StandardActions.java

```diff
@@ -37,6 +37,7 @@ public enum StandardActions implements Action {
     OPEN_URL(Localization.lang("Open URL or DOI"), IconTheme.JabRefIcons.WWW, KeyBinding.OPEN_URL_OR_DOI),
     SEARCH_SHORTSCIENCE(Localization.lang("Search ShortScience")),
     MERGE_WITH_FETCHED_ENTRY(Localization.lang("Get bibliographic data from %0", "DOI/ISBN/...")),
+    MASS_GET_BIBLIOGRAPHIC_DATA(Localization.lang("Get bibliographic data from %0 (fully automated)", "DOI/ISBN/...")),
```

Comment:
Maybe rename `MASS_GET_BIBLIOGRAPHIC_DATA`  to ` MULTI_MERGE_WITH_FETCHED_ENTRY` - to be (somehow) in line with the label and the other code. -- consistent terms from user to code.